### ISSUE TITLE
MacOS Support

### ISSUE THREAD

**Issue by eckelarsson** [hagronnestad/streamdeck-tradfri#1]
Hi, really nice to see someone is giving IKEA TRADFRI some love :)

Just an FYI when adding the plugin on MacOS it do not register, dont know whats wrong or not. But just to let you know.

Great job!

**Comment by olekristian** [comment]
I had this issue on Windows as well. I had to go to task manager to fully close Stream Deck application and start it again for the plugin to show up. 
A restart should have sufficed in both cases.

**Comment by eckelarsson** [comment]
I did that as well on MacOS and it still did't show up.

**Comment by hagronnestad** [comment]
Thanks @eckelarsson :)

> Just an FYI when adding the plugin on MacOS it do not register, dont know whats wrong or not. But just to let you know.

As stated in the README, the plugin has not been tested on MacOS. Did you download the release or build from source? My release is probably only built for Windows. If you can build from source, target and test on MacOS that would be great.

**Comment by hagronnestad** [comment]
@olekristian 

> I had this issue on Windows as well. I had to go to task manager to fully close Stream Deck application and start it again for the plugin to show up. A restart should have sufficed in both cases.

I added a point under "Installation" in the README to make sure to restart the Stream Deck application.

Did you manage to set up your gateway and actions successfully? :)

**Comment by olekristian** [comment]
Yes! Works like a charm after I figured out how to name my bulbs.
Thank you very much.

ons. 19. okt. 2022 kl. 13:19 skrev Hein Andre Grønnestad <
***@***.***>:

**Comment by claband** [comment]
> Thanks @eckelarsson :)
> 
> > Just an FYI when adding the plugin on MacOS it do not register, dont know whats wrong or not. But just to let you know.
> 
> As stated in the README, the plugin has not been tested on MacOS. Did you download the release or build from source? My release is probably only built for Windows. If you can build from source, target and test on MacOS that would be great.

Did someone build for MACOS? I'd like to test this if possible
